### PR TITLE
Added the message at the end of the command /help output

### PR DESCRIPTION
**Pull request by joseotavioq**
A suggestion to solve the issue #2892

### CODE REVIEW

File: src/MSBuild/Resources/Strings.resx [hunk 1]
```diff
@@ -787,6 +787,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_37_DocsLink" UESanitized="false" Visibility="Public">
```
Comment (Forgind) [inline src/MSBuild/Resources/Strings.resx]: We don't use this anymore
```suggestion
  <data name="HelpMessage_37_DocsLink" Visibility="Public">
```

File: src/MSBuild/Resources/Strings.resx [hunk 2]
```diff
@@ -787,6 +787,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
...
+    <value>For more detailed information access the link:
```
Comment (Forgind) [inline src/MSBuild/Resources/Strings.resx]: nit:
```suggestion
    <value>For more detailed information, see:
```

Comment (rainersigwald) [reply src/MSBuild/Resources/Strings.resx]:
```suggestion
    <value>For more detailed information, see https://aka.ms/msbuild/docs
```

? That shortlink points to `https://docs.microsoft.com/visualstudio/msbuild/msbuild` but can be easily moved if we need to later on.

File: src/MSBuild/Resources/Strings.resx (hunk 1 above)
Comment (joseotavioq) [reply src/MSBuild/Resources/Strings.resx]: @Forgind  Done!

File: src/MSBuild/Resources/Strings.resx (hunk 2 above)
Comment (joseotavioq) [reply src/MSBuild/Resources/Strings.resx]: @rainersigwald Done!

**Pull request merged**